### PR TITLE
Chore/split internal external testing

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Description

Testing area is now splitted between `external` and `internal`. Internal has all features enabled + custom wallets. External has all features with custom wallets disabled. 

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

Closes https://linear.app/walletconnect/issue/APKT-668/put-sample-wallets-in-a-separate-page-of-appkit-lab

# Checklist

- [] Code in this PR is covered by automated tests (Unit tests, E2E tests) 
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections

### CODE REVIEW

File: apps/laboratory/src/pages/library/ethers-all-internal.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+const modal = createWeb3Modal({
+  ethersConfig: defaultConfig({
+    metadata: ConstantsUtil.Metadata,
+    defaultChainId: 1,
+    rpcUrl: 'https://cloudflare-eth.com',
+    auth: {
+      socials: ['google', 'x', 'discord', 'apple', 'github']
+    },
+    coinbasePreference: 'smartWalletOnly'
```
Comment (tomiir) [inline apps/laboratory/src/pages/library/ethers-all-internal.tsx:19]: should we remove this so it defaults to `all` ?

**Review by tomiir — APPROVED** [review]
💪

File: apps/laboratory/src/pages/library/ethers-all-internal.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }),
+  chains: EthersConstants.chains,
+  projectId: ConstantsUtil.ProjectId,
+  enableAnalytics: true,
+  metadata: ConstantsUtil.Metadata,
+  termsConditionsUrl: 'https://walletconnect.com/terms',
+  privacyPolicyUrl: 'https://walletconnect.com/privacy',
+  siweConfig,
+  customWallets: ConstantsUtil.CustomWallets
```
Comment (enesozturk) [inline apps/laboratory/src/pages/library/ethers-all-internal.tsx:28]: Should we add all other possible features?
- `tokens`
- `customWallets`
- `allWallets`
- `includeWalletIds`
- `excludeWalletIds`
- `featuredWalletIds`
- `allowUnsupportedChain`

Comment (enesozturk) [reply apps/laboratory/src/pages/library/ethers-all-internal.tsx:28]: Same for wagmi

**Review by enesozturk — APPROVED** [review]

File: apps/laboratory/src/pages/library/ethers-all-internal.tsx (hunk 2 above)
Comment (svenvoskamp) [reply apps/laboratory/src/pages/library/ethers-all-internal.tsx:28]: Not sure if we need to add those, as these are more specific configuration adjustments than actual features. But I agree that we should test these somewhere.

**Pull request merged**